### PR TITLE
GA click tracking

### PR DESCRIPTION
**Pull request by aduggin**
Uses Universal Analytics to track links in different sections of the page so we can track what parts of the page and which links users are clicking.

Adds jasmine and jquery-jasmine to the project

### CODE REVIEW

File: app/assets/javascripts/click-tracker.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+(function ($) {
+  'use strict';
+
+  window.PWPG = window.PWO || {};
```
Comment (stevenwilkin) [inline app/assets/javascripts/click-tracker.js]: Should be `.. = window.PWPG || {}` ?

File: app/assets/javascripts/click-tracker.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      return name || 'homepage';
+    },
+    sendEvent: function(actionLabel) {
+      ga('send', 'event', this.categoryName(), actionLabel, $(this).attr('href'));
+    }
+  };
+
+  // PRIVATE METHODS
+  function bindGlobalHeaderLinks() {
+    $('#global-header a').on( "click", function() {
```
Comment (stevenwilkin) [inline app/assets/javascripts/click-tracker.js]: Mixing single and double quotes here

File: app/assets/javascripts/click-tracker.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+      clickTracker.sendEvent('global header');
+    });
+  }
+
+  function bindMastheadLinks() {
+    $('.l-masthead a').on( "click", function() {
```
Comment (stevenwilkin) [inline app/assets/javascripts/click-tracker.js]: Quotes

Comment (stevenwilkin) [inline app/assets/javascripts/click-tracker.js]: Same for every other occurance of `"click"`

File: vendor/assets/javascripts/jasmine-jquery.js [hunk 4]
```diff
@@ -0,0 +1,832 @@
+/*!
```
Comment (stevenwilkin) [inline vendor/assets/javascripts/jasmine-jquery.js]: This file has been included by accident? It should be pulled into via the asset pipeline from the `jasmine-jquery-rails` gem

File: app/assets/javascripts/application.js [hunk 5]
```diff
@@ -1 +1,5 @@
 //= require jquery/dist/jquery.min.js
+//= require click-tracker.js
+
+
```
Comment (gjvis) [inline app/assets/javascripts/application.js]: Extra empty line - shouldn't JSHint catch this?

File: app/views/guides/show.html.erb [hunk 6]
```diff
@@ -3,7 +3,7 @@
 
 <% content_for :sidebar do %>
   <% if @journey.include?(@guide) %>
-    <div class="link-promo">
+    <div class="link-promo ga-journey-sidebar">
```
Comment (gjvis) [inline app/views/guides/show.html.erb]: I think data attributes are more appropriate than classes for targeting JS behaviour like this

**Pull request merged**